exercises/practice/pig-latin/test/pig_latin_test.dart

```diff
@@ -0,0 +1,130 @@
+import 'package:pig_latin/pig_latin.dart';
+import 'package:test/test.dart';
+
+void main() {
+  final pigLatin = PigLatin();
+
+  group('PigLatin', () {
+    group('ay is added to words that start with vowels', () {
```

Comment:
Depending upon the regex the learner ends up using, or the method of transformation, they might end up removing any extraneous characters/numbers from the output string. So I think we should also consider a test case with numbers in it. The presence of numbers should not affect the transformation. i.e. `'1 apple'` should translate to `'1 appleay'`

Reply:
> A test case for empty string could be introduced.

Again, since Pig Latin is only applied to words, an empty string is outside the scope of the problem. I don't think that the exercise would be more interesting with the addition of an empty string case.

Similarly, numerals are not applicable, because Pig Latin is only about spoken language, not the written encoding. So "1 apple" is the equivalent of "one apple", but it would not really make sense in this context to make people have custom rules hard-coded for numbers.